feconf.py

```diff
@@ -654,6 +654,8 @@ def get_empty_ratings():
     'Puzzles': '#a8554a',
     'Sport': '#893327',
     'Welcome': '#992a2b',
+
+    'Default': '#a33f40',
```

Comment:
Don't declare a new category. There is a default color specified somewhere already; pass that to the frontend instead if you need it.